### PR TITLE
Feature/update instructions

### PR DESCRIPTION
**Pull request by ThornM9**

### CODE REVIEW

File: programs/data-credits/src/instructions/update_in_use_data_credits_v0.rs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+use anchor_lang::prelude::*;
+
+#[derive(AnchorSerialize, AnchorDeserialize, Clone, Default)]
+pub struct UpdateInUseDataCreditsArgsV0 {
+  new_owner: Pubkey,
+}
+
+#[derive(Accounts)]
+#[instruction(args: UpdateInUseDataCreditsArgsV0)]
+pub struct UpdateInUseDataCreditsV0<'info> {
```
Comment (ChewingGlass) [inline programs/data-credits/src/instructions/update_in_use_data_credits_v0.rs:11]: Just rm this for now, we don't need this endpoint. In use data credits are about to change anyway

**Review by ChewingGlass — CHANGES_REQUESTED** [review]

File: programs/circuit-breaker/src/instructions/update_account_windowed_breaker_v0.rs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+#[derive(Accounts)]
+pub struct UpdateAccountWindowedBreakerV0<'info> {
+  #[account(
+    mut,
+    seeds = ["account_windowed_breaker".as_bytes(), token_account.key().as_ref()],
+    bump,
+    has_one = authority,
+  )]
+  pub circuit_breaker: Box<Account<'info, AccountWindowedCircuitBreakerV0>>,
+  pub token_account: Box<Account<'info, TokenAccount>>,
```
[the remark this answers is not in the document]
Comment (ThornM9) [reply programs/circuit-breaker/src/instructions/update_account_windowed_breaker_v0.rs:21]: Fixed

File: programs/circuit-breaker/src/instructions/update_mint_windowed_breaker_v0.rs [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+pub struct UpdateMintWindowedBreakerV0<'info> {
+  pub authority: Signer<'info>,
+  #[account(
+    mut,
+    seeds = ["mint_windowed_breaker".as_bytes(), mint.key().as_ref()],
+    bump,
+    has_one = authority,
+  )]
+  pub circuit_breaker: Box<Account<'info, MintWindowedCircuitBreakerV0>>,
+  pub mint: Box<Account<'info, Mint>>,
```
[the remark this answers is not in the document]
Comment (ThornM9) [reply programs/circuit-breaker/src/instructions/update_mint_windowed_breaker_v0.rs:22]: Fixed

File: programs/data-credits/src/instructions/update_in_use_data_credits_v0.rs (hunk 1 above)
Comment (ThornM9) [reply programs/data-credits/src/instructions/update_in_use_data_credits_v0.rs:11]: Done

File: programs/helium-entity-manager/src/instructions/update_hotspot_issuer_v0.rs [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+}
+
+#[derive(Accounts)]
+#[instruction(args: UpdateHotspotIssuerArgsV0)]
+pub struct UpdateHotspotIssuerV0<'info> {
+  #[account(
+    mut,
+    has_one = authority,
+  )]
+  pub hotspot_issuer: Box<Account<'info, HotspotIssuerV0>>,
```
[the remark this answers is not in the document]
Comment (ThornM9) [reply programs/helium-entity-manager/src/instructions/update_hotspot_issuer_v0.rs:17]: There's potentially multiple issuers for a config, and it should be permissionless to add a new issuer.

**Review by ChewingGlass — APPROVED** [review]

**Pull request merged**